Stellar-contract.x

```diff
@@ -128,6 +129,16 @@ enum SCHostStorageErrorCode
     HOST_STORAGE_GET_ON_DELETED_KEY = 5
 };
 
+enum SCHostAuthErrorCode
+{
+    HOST_AUTH_UNKNOWN_ERROR = 0,
+    HOST_AUTH_WRONG_NONCE = 1,
```

Comment:
Can we rename this to just `NONCE_ERROR` in order to not touch it anymore when we change the nonce approach?

Reply:
Done